pkgs/kde/gear/kio-extras/default.nix

```
@@ -0,0 +1,43 @@
+{
+  mkKdeDerivation,
+  qt5compat,
+  qtsvg,
+  pkg-config,
+  libkexiv2,
+  samba,
+  libssh,
+  libmtp,
+  libimobiledevice,
+  gperf,
+  libtirpc,
+  openexr,
+  taglib,
+  shared-mime-info,
+  libappimage,
+  kio,
+}:
```

```suggestion
  shared-mime-info,
  libappimage,
  kio,
  libX11,
  libXcursor,
}:
```

See https://github.com/NixOS/nixpkgs/pull/243816